src/XR/webXRFeaturesManager.ts

```diff
@@ -77,6 +77,10 @@ export class WebXRFeatureName {
      * The name of the teleportation feature
      */
     public static TELEPORTATION = "xr-controller-teleportation";
+    /**
+     * The name of the feature points feature.
+     */
+    public static FEATURE_POINTS = "xr-feature-points";

```

Comment:
Presumably this (and others) should be `readonly`?

Reply:
I think that's probably true I was following what was already in place here @RaananW thoughts?

Reply:
`readonly` makes sense, of course. You can change the rest to readonly as well.